Sources/Operation/OriginateAccountOperation.swift

```diff
@@ -5,23 +5,51 @@ import Foundation
 /** An operation that originates a new KT1 account. */
 public class OriginateAccountOperation: AbstractOperation {
   let managerPublicKeyHash: String
+  let contractCode: ContractCode?
```

Comment:
Should these properties both be private?

Reply:
Done.